config.json

```diff
@@ -451,7 +451,15 @@
       "topics": [
 
       ]
-    }
+    },
+    {
+      "uuid": "589b763c-2b3a-11e8-b467-0ed5f89f718b",
+      "slug": "pascals-triangle",
+      "core": false,
+      "unlocked_by": null,
+      "difficulty": 6,
```

Comment:
Can you put this somewhere near other exercises with difficulty `6`?

Reply:
I just did :) Could you merge this?

Reply:
@NobbZ Could you merge this PR?

Reply:
Sorry for the delay...

Its quite busy here ;)

I'll merge.